### ISSUE TITLE
Apache Flink Support

### ISSUE THREAD

**Issue by Madhu1512** [elastic/elasticsearch-hadoop#573]
Is there any plan in future to provide elasticsearch-hadoop connector for Apache flink?

**Comment by costin** [comment]
No, not at this point. Keeping the features in sync across multiple integrations is quite a challenge and adding a new one is a decision that we don't take lightly especially from the maintenance POV; it's easy to create a first draft, not so to make it fully featured and _tested_.
Let alone keeping it in sync with the various releases...
